### DISCUSSION TITLE
MHI2 broken after patch

### DISCUSSION THREAD

**Discussion by TTTuv95** [Mr-MIBonk/M.I.B._More-Incredible-Bash#253] [Q&A]
Hello, 
I updated MHI2 from P3250 to K3342 and everything was fine i then ran patch it seemed to be fine but after it there was no sound i tried to run backup to restore files but mmi restared mid backup and now mmi is bricked it boots for 5mins and after boot screen is just stuck nothing works and it reboot itself. Tried to put swdlautorun in sd root and put sd card with firmware update to SD readet but that doesn’t work also. Any solutions how to fix this ? Or do i need now to take out unit and give it to someone for manual fix ?

**Comment by dootzord** [comment]
Would love to know if you resolved this? I had a 'MOST not available' error on update of my MHI2 unit and now it only boots to a radio mode and I cannot enter the developer menu to resolve it or update

**Comment by harman-f** [comment]
> Hello,
> I updated MHI2 from P3250 to K3342 and everything was fine i then ran patch it seemed to be fine but after it there was no sound i tried to run backup to restore files but mmi restared mid backup and now mmi is bricked it boots for 5mins and after boot screen is just stuck nothing works and it reboot itself. Tried to put swdlautorun in sd root and put sd card with firmware update to SD readet but that doesn’t work also. Any solutions how to fix this ? Or do i need now to take out unit and give it to someone for manual fix ?

Your LOG file would help, if unit is still a brick.

**Reply by dootzord** [reply]
Cannot get to any of those screens away from the radio screen...

**Reply by TTTuv95** [reply]
Hello,
Unit was recovered via UART, still don't know what exactly caused the problem and i no longer have log files from that car. Seems like some kind of error ocurred during Patching, never had this happen before.

**Reply by kcisoul** [reply]
I'm encountering a similar issue. Could you guide me on how to perform a restoration? I'm not familiar with what UART signifies.
